__tests__/session.spec.ts

```diff
@@ -159,6 +159,52 @@ describe('session.ts', () => {
       jest.replaceProperty(envVariables, 'WORKOS_REDIRECT_URI', originalWorkosRedirectUri);
     });
 
+    it('should throw an error if the cookie password is not set', async () => {
+      const originalWorkosCookiePassword = envVariables.WORKOS_COOKIE_PASSWORD;
+
+      jest.replaceProperty(envVariables, 'WORKOS_COOKIE_PASSWORD', '');
+
+      await expect(async () => {
+        await updateSession(
+          new NextRequest(new URL('http://example.com')),
+          false,
+          {
+            enabled: false,
+            unauthenticatedPaths: [],
+          },
+          '',
+          [],
+        );
+      }).rejects.toThrow(
+        'You must provide a valid cookie password that is at least 32 characters in the environment variables.',
+      );
+
+      jest.replaceProperty(envVariables, 'WORKOS_COOKIE_PASSWORD', originalWorkosCookiePassword);
```

Comment:
If you set `restoreMocks` to true in the Jest config then properties like this will automatically be restored between tests without a need to keep track of original values like this. I think it's a good default behavior to go with.